store/storetest/channel_store.go

```diff
@@ -911,7 +911,7 @@ func testChannelMemberStore(t *testing.T, ss store.Store) {
 		t.Fatal("should have saved 2 members")
 	}
 
-	store.Must(ss.Channel().RemoveMember(o2.ChannelId, o2.UserId))
+	ss.Channel().RemoveMember(o2.ChannelId, o2.UserId)
```

Comment:
To be consistent with the previous behavior I miss to handle the error properly here with a `require.Nil(t, err)`

Reply:
Fixed.